cost/superseded_instance_remediation/README.md

```diff
@@ -0,0 +1,38 @@
+## Superseded Instance Types Remediation Policy
+
+### What it does
+
+This Policy Template is used to automatically resize instances based on user-defined standards.  For example, the Policy uses a Disallowed Instance Type and its corresponding Superseded Instance Type.  If any Instances are found using the Disallowed type, they will be included in the corresponing Policy Incident.  Upon approval, Cloud Workflow will resize the Instance to the Superseded type. 
+
+### Prerequesites
+- The following RightScale Credentials
+  - Cloud Management - `Actor`
+  - Cloud Management - `Observer`
+  - Governance -  `policy_designer`
```

Comment:
Let's start excluding the `policy_*` roles from docs. They are more or less inferred.